packages/__tests__/3-runtime-html/styles.spec.ts

```diff
@@ -197,35 +181,41 @@ describe('Styles', function () {
       assert.equal(fake.wasCalled, true);
     });
 
-    // it('projector applies styles during projection', function () {
-    //   const ctx = TestContext.createHTMLTestContext();
-    //   const host = ctx.createElement('foo-bar');
-    //   const FooBar = CustomElement.define(
-    //     {
-    //       name: 'foo-bar',
-    //       shadowOptions: { mode: 'open' }
-    //     }
-    //   );
-    //   const css = '.my-class { color: red }';
-    //   const context = ctx.container.createChild();
-    //   context.register(
-    //     Registration.instance(
-    //       IShadowDOMStyles,
-    //       new StyleElementStyles(ctx.dom, [css], null)
-    //     )
-    //   );
-
-    //   const component = new FooBar();
-    //   const controller = Controller.forCustomElement(component as ICustomElementViewModel<HTMLElement>, ctx.container.get(ILifecycle), host, ctx.container, void 0);
-
-    //   const seq = { appendTo() { return; } };
-    //   const projector = controller.projector;
-
-    //   projector.project(seq as any);
-
-    //   const root = projector.provideEncapsulationSource() as unknown as ShadowRoot;
-    //   assert.strictEqual(root.firstElementChild.innerHTML, css);
-    // });
+    it.skip('projector applies styles during projection', function () {
+      const ctx = TestContext.createHTMLTestContext();
+      const host = ctx.createElement('foo-bar');
+      const FooBar = CustomElement.define(
+        {
+          name: 'foo-bar',
+          shadowOptions: { mode: 'open' }
+        }
+      );
+      const css = '.my-class { color: red }';
+      const context = ctx.container.createChild();
+      context.register(
+        Registration.instance(
+          IShadowDOMStyles,
+          new StyleElementStyles(ctx.dom, [css], null)
+        )
+      );
+
+      const component = new FooBar();
+      const controller = Controller.forCustomElement(
+        component as ICustomElementViewModel<HTMLElement>,
+        ctx.container.get(ILifecycle),
```

Comment:
Convenience shorthand already exists

```suggestion
        ctx.lifecycle,
```

Reply:
Fixed.